### PR TITLE
Tournament Settings

### PR DESCRIPTION
**Pull request by Senryoku**

### CODE REVIEW

File: src/Session.ts [hunk 1]
```diff
@@ -74,6 +73,25 @@ import { isSomeEnum } from "./TypeChecks.js";
 import { askColors, choosePlayer } from "./Conspiracy.js";
 import { InProduction, TestingOnly } from "./Context.js";
 
+// Tournament timer depending on the number of remaining cards in a pack.
+const TournamentTimer = [
+	0,
+	5, // 1 card remaining. Should be immediate
+	5,
```
Comment (tooomm) [inline src/Session.ts:80]: When testing this, I had 5s to pick the last card in a 15 cards booster pack when using tournament timer. It was not instant.

**Comment by tooomm** [comment]
Just came around to finally test this a bit more in detail.
Awesome that you made it happen! Love it. 🚀 

I realized that the review phase does not increase with each pack. Regarding the [linked rules](https://blogs.magicjudges.org/rules/mtr-appendix-b/) it should increase from 60 to 90 after the second pack to accompany for the increased amount of cards and still keep it tight..
But since you allow to set a custom value, maybe add another 20s after each pack? Or use a 1.33 multiplier (instead 50%) to go from e.g. 45s to 60s?

File: src/Session.ts (hunk 1 above)
Comment (Senryoku) [reply src/Session.ts:80]: Yeah, that's what I meant with this comment, it should be instant per the rules, but I choose to keep it at 5sec for a better user experience.

**Comment by Senryoku** [comment]
Yep, I missed that rule.
f3a9c07 increases the timer by 50% (so +30sec for a 60sec timer, as stated in the rules) after each subsequent pack (up to 2 times, I don't think it makes sense pass that).

**Comment by tooomm** [comment]
Sure, 1.5 also works!
I suggested a bit lower to account for the digital experience and that it takes less time to review the pool online than flicking through a real pile of cards.

**Pull request merged**